docs/cluster-management/running-locally.md

```diff
@@ -61,6 +61,38 @@ To start up the SD-in-a-Box, execute the following command
 $ docker-compose -p screwdriver up
 ```
 
+### Volume-Mounted Source Code
+
+You can choose to replace a component with a local copy. This is incredibly helpful if you're trying to implement an update to a service and want to see how it impacts the entire cluster.
+
+Modify the `docker-compose.yaml`, targeting the component you would like to replace. In the following snippet, we replace the API with a local source.
```

Comment:
Also remove `node_modules/` from `.dockerigore`

@minz1027 and I have been doing it this way...